azuredevops/internal/service/serviceendpoint/resource_serviceendpoint_azurerm_test.go

```diff
@@ -125,6 +125,128 @@ var azurermTestServiceEndpointsAzureRM = []serviceendpoint.ServiceEndpoint{
 			},
 		},
 	},
+	{
+		Authorization: &serviceendpoint.EndpointAuthorization{
+			Parameters: &map[string]string{
+				"tenantid": "aba07645-051c-44b4-b806-c34d33f3dcd1", //fake value
+			},
+			Scheme: converter.String("ManagedServiceIdentity"),
+		},
+		Data: &map[string]string{
+			"environment":      "AzureCloud",
+			"scopeLevel":       "Subscription",
+			"subscriptionId":   "42125daf-72fd-417c-9ea7-080690625ad3", //fake value
+			"subscriptionName": "SUBSCRIPTION_TEST",
+		},
+		Id:    &azurermTestServiceEndpointAzureRMID,
+		Name:  converter.String("_AZURERM_UNIT_TEST_CONN_NAME"),
+		Owner: converter.String("library"), // Supported values are "library", "agentcloud"
+		Type:  converter.String("azurerm"),
+		Url:   converter.String("https://management.azure.com/"),
+		ServiceEndpointProjectReferences: &[]serviceendpoint.ServiceEndpointProjectReference{
+			{
+				ProjectReference: &serviceendpoint.ProjectReference{
+					Id: azurermTestServiceEndpointAzureRMProjectID,
+				},
+				Name:        converter.String("_AZURERM_UNIT_TEST_CONN_NAME"),
+				Description: converter.String("_AZURERM_UNIT_TEST_CONN_DESCRIPTION"),
+			},
+		},
+	},
+	{
+		Authorization: &serviceendpoint.EndpointAuthorization{
+			Parameters: &map[string]string{
+				"tenantid":           "aba07645-051c-44b4-b806-c34d33f3dcd1", //fake value
+				"serviceprincipalid": "bba07645-051c-44b4-b806-c34d33f3dcd2", //fake value
+			},
+			Scheme: converter.String("WorkloadIdentityFederation"),
+		},
+		Data: &map[string]string{
+			"environment":      "AzureCloud",
+			"scopeLevel":       "Subscription",
+			"subscriptionId":   "42125daf-72fd-417c-9ea7-080690625ad3", //fake value
+			"subscriptionName": "SUBSCRIPTION_TEST",
+			"creationMode":     "Manual",
+		},
+		Id:    &azurermTestServiceEndpointAzureRMID,
+		Name:  converter.String("_AZURERM_UNIT_TEST_CONN_NAME"),
+		Owner: converter.String("library"), // Supported values are "library", "agentcloud"
+		Type:  converter.String("azurerm"),
+		Url:   converter.String("https://management.azure.com/"),
+		ServiceEndpointProjectReferences: &[]serviceendpoint.ServiceEndpointProjectReference{
+			{
+				ProjectReference: &serviceendpoint.ProjectReference{
+					Id:   azurermTestServiceEndpointAzureRMProjectID,
+					Name: converter.String("doesntmatter"),
```

Comment:
I'd recommend not setting it at all. For a time being it will continue failing, but once our fix is rolled out it will work without issues.

Reply:
If I can run the acceptance tests successfully before this gets merged I will remove it. Otherwise I will raise a separate PR to remove it later.